dataland-frontend/src/components/general/JoinDatalandButton.vue

```diff
@@ -23,7 +23,8 @@ export default {
       this.getKeycloakPromise()
         .then((keycloak) => {
           if (!keycloak.authenticated) {
-            const url = keycloak.createRegisterUrl({ redirectUri: "/companies" });
+            let baseUrl = window.location.origin;
+            const url = keycloak.createRegisterUrl({ redirectUri: `${baseUrl}/companies` });
```

Comment:
Warum muss man jetzt die baseUrl explizit setzen?
Vorher war die denke ich bei KeyCloak in den configs hinterlegt, weshalb redirect auf "/companies" ausreichend war.